tests/open_test.py

```diff
@@ -40,15 +40,24 @@ def test_open():
     _cleanup_generated_files()
 
     # convert can also be a path
-    target = os.path.join(path, 'data', 'convert.hdf5')
+    target_path = os.path.join(path, 'data', 'output')
+    target = os.path.join(target_path, 'convert.hdf5')
```

Comment:
BTW there are more tests for CSV operations in the `from_csv_test.py`. But the test for the `convert=file_path` is passing a `filename_or_buffer=file_descriptor`, so it works :/